src/FSLibrary/StellarKubeSpecs.fs

```diff
@@ -263,6 +263,7 @@ let CoreContainerForCommand
         V1EnvVar(name = CfgVal.asanOptionsEnvVarName, value = CfgVal.asanOptionsEnvVarValue)
 
     let cfgWords = cfgFileArgs configOpt MainCoreContainer
+    let cfgWords = Array.append cfgWords [| ShWord.OfStr "--console" |]
```

Comment:
mm this isn't related to the config file though, right? so probably needs its own variable that gets appended to cmdWords

Reply:
Good point. Fixed.